math/src/polynom/univariate.rs

```diff
@@ -0,0 +1,268 @@
+use alloc::vec::Vec;
+
+use utils::{ByteReader, ByteWriter, Deserializable, DeserializationError, Serializable};
+
+use crate::{batch_inversion, polynom, FieldElement};
+
+// UNIVARIATE POLYNOMIAL (EVALUATION FORM)
+// ================================================================================================
+
+/// The coefficients of a univariate polynomial of degree n with the linear term coefficient
+/// omitted.
+#[derive(Clone, Debug)]
+pub struct UnivariatePolyCoef<E: FieldElement> {
+    pub coefficients: Vec<E>,
+}
```

Comment:
Since these are used internally by the sum-check protocol, I think it makes sense to keep the newtype-based implementation. Moreover, performance considerations should not be an issue as the polynomials we will be dealing with are of low degree e.g., definitely less than 10.

I update the type to the suggestion above.